appinfo/info.xml

```diff
@@ -9,13 +9,12 @@
 	<name>End-to-End Encryption</name>
 	<summary>End-to-end encryption endpoint</summary>
 	<description><![CDATA[
-Provides the necessary endpoint to enable end-to-end encryption.
-
-**Notice:** E2EE is currently not compatible to be used together with server-side encryption
+This app provides all the necessary APIs to implement End-to-End encryption on the client side.
+Additionally it makes sure that End-to-End encrypted files are not accessible via the web interface and other WebDAV clients and implemented Secure FileDrop.
```

Comment:
```suggestion
Additionally it makes sure that End-to-End encrypted files are not accessible via the web interface and other WebDAV clients and implements Secure FileDrop.
```
The sentence is still a bit weird with the two "and" though :thinking:

Reply:
Changed it to:

> Additionally it implements Secure FileDrop and makes sure that End-to-End encrypted files are neither accessible via the web interface nor other WebDAV clients.